To: vim_dev@googlegroups.com
Subject: Patch 7.3.147
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.147 (after 7.3.143)
Problem:    Can't build on HP-UX.
Solution:   Remove an unnecessary backslash. (John Marriott)
Files:	    src/Makefile


*** ../vim-7.3.146/src/Makefile	2011-03-22 18:10:34.000000000 +0100
--- src/Makefile	2011-04-01 13:00:58.000000000 +0200
***************
*** 1565,1571 ****
  
  OBJ = $(OBJ_COMMON) \
  	objects/main.o \
! 	objects/memfile.o \
  
  MEMFILE_TEST_OBJ = $(OBJ_COMMON) \
          objects/memfile_test.o
--- 1565,1571 ----
  
  OBJ = $(OBJ_COMMON) \
  	objects/main.o \
! 	objects/memfile.o
  
  MEMFILE_TEST_OBJ = $(OBJ_COMMON) \
          objects/memfile_test.o
*** ../vim-7.3.146/src/version.c	2011-03-27 16:03:09.000000000 +0200
--- src/version.c	2011-04-01 13:05:18.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     147,
  /**/

-- 
DENNIS: You can't expect to wield supreme executive power just 'cause some
        watery tart threw a sword at you!
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
